packages/next/src/client/components/app-router.tsx

```diff
@@ -414,7 +414,6 @@ function Router({
           })
         })
       },
-      // @ts-ignore we don't want to expose this method at all
```

Comment:
Instead of `@ts-ignore` here, we can implement the type and mark it with `@internal` so it still shows up for us in the monorepo.